contracts/src/v0.8/ccip/pools/GHO/UpgradeableTokenPool.sol

```diff
@@ -317,4 +328,23 @@ abstract contract UpgradeableTokenPool is IPool, OwnerIsCreator, IERC165 {
     if (IARM(i_armProxy).isCursed()) revert BadARMSignal();
     _;
   }
+
+  /// @notice Getter for proxy pool address.
+  /// @return proxyPool The proxy pool address for the given remoteChainSelector
+  function getProxyPool() public view returns (address proxyPool) {
+    assembly ("memory-safe") {
+      proxyPool := shr(96, shl(96, sload(PROXY_POOL_SLOT)))
+    }
+  }
+
+  /// @notice Setter for proxy pool address, only callable by the DAO.
+  /// @dev This router is currently set for the Eth/Arb lane, and this pool is not expected
+  /// to support any other lanes in the future - hence can be stored agnostic to chain selector.
```

Comment:
not sure I follow. This will work even if used for any other chain

Reply:
true! rm-ed [623c10c](https://github.com/aave/ccip/pull/18/commits/623c10c203d5187eeb82a998ce3c01ea94bce6d9)